docs/file_explorer.md

```diff
@@ -39,23 +39,33 @@ Here are the basic operations of managing a Workspace. The letters in bold below
 - **C.** Delete a Workspace  <br>
 -  **D.** Download all Workspaces This will create a .zip file with all the files of all the Workspaces.  The zip file will have a folder called **.workspaces** that will contain a folder of each workspace.  Depending on your OS, you may need to change the preferences on .workspaces folder to make it visible.  <br>
 - **E.** Upload the Workspaces backup made from the previous icon. <br>
-- **K.** Choose a Workspace 
+- **F.** Clone a repo from Git repository <br>
+- **G.** Link to documentation <br>
+- **J.** Publish the Workspace to a GIST <br>
+- **L.** Choose a Workspace 
 ### File Manipulation
 The letters in bold below refer to the labels in fig. 1.
 
-- **G.** Create a file  <br>
-- **H.** Create a folder  <br>
-- **I.** Publish the Workspace to a GIST <br>
-- **J.** Load a local file into the current Workspace<br>
+- **H.** Create a file  <br>
+- **I.** Create a folder  <br>
+- **K.** Load a local file into the current Workspace<br>
 
 ## Workspaces with Templates
 When you create a new Workspace, a modal comes up where you choose which template of files to include.
 
 ![](images/a-workspace-templates.png)
 
+## Workspaces & cloning a repo
+When you click the **clone** icon (**F.** in fig.1), you'll be asked for the url of the repo.  A new workspace will be created that will contain the cloned repo.  To manage the Git repo, go to the DGIT plugin.
+
+## Workspaces with a Git repo
+Workspaces with an associated Git will have the Git icon next to them in the Workspaces select box.
+
+![](images/a-fe-select-git.png)
+
 ## Creating new files
 
-There are 2 ways of creating files.  The first is to click on the new file icon (**G.** in fig.1), an input for a new the file’s name will appear in the **File Explorer**. Once a name is entered, the new empty file will open in the Editor.
+There are 2 ways of creating files.  The first is to click on the new file icon (**H.** in fig.1), an input for a new the file’s name will appear in the **File Explorer**. Once a name is entered, the new empty file will open in the Editor.
```

Comment:
`the` should be removed from `an input for a new the file’s`

Reply:
You can also add that file name is entered without any file extension, `.sol` will be assumed by default

Reply:
Nice addition